src/lib.ts

```diff
@@ -79,30 +81,29 @@ export default async function isMergable(opts: Options): Promise<Result> {
 	let pendingChecks: { name: string }[] = []
 	let missingChecks: { name: string }[] = []
 
-	let latestReviews: Map<
-		string,
-		Octokit.PullsListReviewsResponseItem
-	> = new Map()
+	let latestReviews: Map<string, typeof reviews.data[0]> = new Map()
 
 	for (let review of reviews.data) {
-		let prev = latestReviews.get(review.user.login)
+		const userLogin = review.user ? review.user.login : "unknownReviewer"
```

Comment:
`review.user?.login ?? "unknownReviwer"`

Reply:
😅 I was being lazy because I didn't want to update the babel configs.